.github/workflows/test-aot-cli.yml

```diff
@@ -5,41 +5,26 @@ on:
     branches: [ master ]
   pull_request:
     branches: [ master ]
-
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.event_name }}
+  cancel-in-progress: true
 jobs:
   build:
     strategy:
       matrix:
-        runs-on-container:
-          - ubuntu:22.04
-          - fedora:39
+        container:
+          - ubuntu-2204
+          - fedora-39
     runs-on: ubuntu-latest
     container: 
-      image: ${{matrix.runs-on-container}}
+      image: "manjusakalza/bpftime-base-image:${{matrix.container}}"
```

Comment:
I think these images should be owned by account eunomia-bpf

Reply:
Yes, I will update the image and add a new workflow to build the image when we reach consensus

Reply:
> Yes, I will update the image and add a new workflow to build the image when we reach consensus

What about moving these images to eunomia-bpf now? Do you have permission to publish image?